.travis.yml

```diff
@@ -80,6 +81,10 @@ install:
     docker run --rm dockcross/linux-armv7 > ./dockcross-linux-armv7;
     chmod +x ./dockcross-linux-armv7;
   fi
+- if [[ "${BUILD_TARGET}" = "osx_build" ]] || [[ "${BUILD_TARGET}" = "ios_build" ]] || [[ "${BUILD_TARGET}" = "ios_simulator_build" ]]; then
+    brew install ccache;
+    export PATH="/usr/local/opt/ccache/libexec:$PATH";
```

Comment:
so that's a travis path that stays between builds?

Reply:
According to the docs, yes. I doubt it though.
https://docs.travis-ci.com/user/caching/#ccache-cache